content/hardware/required/autopilot/index.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Autopilot (Flight Controller) Board"
+description = "Processes the pilot input and sensor data, and controls the lights and actuators on the vehicle."
+date = 2022-10-11T17:33:19+11:00
+template = "docs/page.html"
+sort_by = "weight"
+weight = 0
+draft = false
+[extra]
+lead = ""
+toc = true
+top = false
++++
+
+In the ArduSub control system, the autopilot board (also known as a flight controller) is the component which the ArduSub firmware is loaded onto. The autopilot processes the pilot input and sensor data, and controls the motors, lights, servos, and relays on the vehicle.
+
+Although there are many different autopilot boards available, here are the general characteristics:
+* Ability to load any ArduPilot binary firmware file (Copter, Plane, Rover/Boat, Sub)
+* Manual control and semi-autonomous/stablilised "flight" modes
```

Comment:
should we really omit the full autonomous thing? I think it should be here, since the scope is more ardupilot than ardusub